addons/load-extensions/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Auto-show editor extensions",
```

Comment:
Auto-load, perhaps?

Reply:
The PR that added this addon was titled "Auto-add extensions", which is also a good name (even without the word "editor": shorter names are better).